helm/templates/superdesk-deployment.yaml

```diff
@@ -21,33 +21,45 @@ spec:
                 operator: NotIn
                 values:
                 - lb
-            - matchExpressions:
-              - key: instance/type
-                operator: NotIn
-                values:
                 - storage
-            - matchExpressions:
-              - key: instance/type
-                operator: NotIn
-                values:
                 - database
       containers:
         - name: client
           image: "{{ .Values.superdesk.image }}:{{ .Values.superdesk.tag }}"
           resources:
             limits:
               cpu: "8"
-              memory: 16Gi
+              memory: 32Gi
```

Comment:
I'm not sure what this increase should change. If you look at the graphs, the container scratches the 10Gi memory line in the last month. It's not reaching the old limit of 16Gi at all, so what should double the limit do for the container? 
![Screen Shot 2020-07-10 at 10 27 27](https://user-images.githubusercontent.com/1640534/87133733-07ce4e80-c298-11ea-806c-c3a07de693a8.png)

Reply:
It's about a out of memory error on some mongodb sort operations (unfortunately we've logs back since some days)

Reply:
But that sounds like a mongo issue. if you increase the memory of the superdesk deployment it won't solve OOM-Issues on the mongo parts.

Reply:
The increased memory value was based on the prod system, there's currently a limit of 20GB set, which is almost consumed  by MongoDB.